rouana/domain.yml

```diff
@@ -586,9 +660,17 @@ templates:
           - "Se você for uma pessoa física você precisa atuar na área cultural."
           - "Caso você seja uma pessoa jurídica pública tem que atuar na área de administração indireta ou uma pessoa jurídica privada com ou sem fins lucrativos."
   utter_geralIntroduzirContextoNovaProposta:
-    - text:
-        - - "Que bom! É sempre bom trabalhar em novos projetos culturais"
-          - "Se você quiser, posso tirar algumas de suas dúvidas para te ajudar neste novo projeto"
+    - text: |
```

Comment:
A intenção aqui é ser dois balões de mensagens separados? Ou apenas um com o duas linhas?

Se for dois balões a estrutura deve ser:
```yaml
- text: |
  Que bom! Vamos focar no seu projeto!

  Você tem dúvidas sobre ele?
```

Reply:
A intenção é dois balões, um em seguida do outro, vou modificar, obrigada pela dica (: